File: Gym/app/src/main/java/com/example/gym/models/Card.kt
package com.example.gym.models

import com.google.gson.annotations.SerializedName
import java.math.BigInteger

data class Card (

    @SerializedName("idCard") val idCard : Int,
    @SerializedName("name") val name : String,
    @SerializedName("period") var period : BigInteger,
    @SerializedName("price") val price : Int,
    @SerializedName("image") val image : String
)

File: Gym_.NET-master/Gym.API/Domain/Models/Card.cs
using System.Numerics;
using System.Collections.Generic;

namespace Gym.API.Domain.Models
{

    public class Card
    {
        public int IdCard { get; set; }
        public string Name { get; set; }

        public long Period { get; set; }
        public decimal Price { get; set; }
        public string Image {get; set;}

        public IList<Client> Client {get;set;} = new List<Client>();

        public IList<ServicesCard> ServicesCard {get;set;} = new List<ServicesCard>();
    }
}